README.md

```diff
@@ -590,7 +588,34 @@ encountered.
 
 ### 5. Reporting Issues
 
-If you are having problems with HarbourBridge, please [submit an
-issue](https://github.com/cloudspannerecosystem/harbourbridge/issues).
+## Known Issues
+Please refer to the [issues section](https://github.com/cloudspannerecosystem/harbourbridge/issues)
+ on Github for a full list of known issues.
+
+### Schema Conversion
+
+- Loading dump files from SQL Server and DynamoDB is not supported
```

Comment:
Should we include Oracle also here, because dump file is not supported for Oracle too?

Reply:
Done